### PR TITLE
(CL) Make Increase Liquidity Modal Mobile Responsive

### PR DESCRIPTION
**Pull request by JoseRFelix**

### CODE REVIEW

File: packages/web/modals/increase-concentrated-liquidity.tsx [hunk 1]
```diff
@@ -120,12 +122,35 @@ export const IncreaseConcentratedLiquidityModal: FunctionComponent<
...
+                id="close-thin"
+                className="text-wosmongton-400 hover:text-wosmongton-100"
+                height={24}
+                width={24}
+              />
+            }
+            onClick={props.onRequestClose}
+          />
+        </div>
+      </div>
```
Comment (jonator) [inline packages/web/modals/increase-concentrated-liquidity.tsx:150]: i think the basemodal provides a close button for use if we simply set hideclosebutton=false

File: packages/web/components/cards/my-position/status.tsx [hunk 2]
```diff
@@ -13,6 +13,7 @@ export const MyPositionStatus: FunctionComponent<{
   lowerPrice: Dec;
   upperPrice: Dec;
   negative?: boolean;
+  className?: string;
```
Comment (jonator) [inline packages/web/components/cards/my-position/status.tsx:16]: giga nit: we have `CustomClasses` type that adds this

**Review by jonator — COMMENTED** [review]
Nits

**Review by jonator — CHANGES_REQUESTED** [review]
See some issues with arrangement of +/- buttons on iPhone
Screenshot 2023-06-10 at 12 28 50 PM

**Comment by JoseRFelix** [comment]
> See some issues with arrangement of +/- buttons on iPhone

Interesting I'm getting the correct size on the first load. It could be an issue with `onresize` event. Can you confirm if it appears correctly when you load it for the first time on your side?

image

**Review by jonator — APPROVED** [review]

**Pull request merged**